java/org.travlyn.server/src/main/java/org/travlyn/server/service/TravlynService.java

```diff
@@ -378,7 +433,9 @@ public void updateTrip(Trip trip) throws NoResultException {
                 .getResultList();
         ArrayList<Trip> trips = new ArrayList<>();
         for (TripEntity entity : result) {
-            trips.add(entity.toDataTransferObject());
+            if (!entity.isPrivate() || userId == getUserId()) {
```

Comment:
Change can be undone if we remove userId param here, like you have done in trip api.

Reply:
Maybe we want to fetch another users public trips